client/service/src/client/call_executor.rs

```diff
@@ -223,7 +223,7 @@ where
 				let state_runtime_code = sp_state_machine::backend::BackendRuntimeCode::new(trie_state);
 				// It is important to extract the runtime code here before we create the proof
 				// recorder.
-				let runtime_code = state_runtime_code.runtime_code()
+				let runtime_code = state_runtime_code.runtime_code(execution_manager.context)
```

Comment:
These are a bit unfortunate to hardcode; I could technically pass around a `ExecutionContext` or similar to all of these functions, but not sure if it is worth all the refactoring that it will impose.